doc/api/process.md

```diff
@@ -231,7 +231,7 @@ For example:
 
 ```js
 process.on('unhandledRejection', (reason, p) => {
-  console.log('Unhandled Rejection at: Promise', p, 'reason:', reason);
+  console.log('Unhandled Rejection at:', p, 'reason:', reason);
```

Comment:
`Promise` is already present now in the output as a prefix for `p`. Without this fix the output will be:
```
Unhandled Rejection at: Promise Promise {
```